src/network/networkd-dhcp4.c

```diff
@@ -454,19 +488,15 @@ static int dhcp4_request_static_routes(Link *link, struct in_addr *ret_default_g
         return classless_route;
 }
 
-static int dhcp4_request_gateway(Link *link, struct in_addr *ret_gw) {
+static int dhcp4_request_gateway(Link *link, struct in_addr *gw) {
```

Comment:
why is the "ret_" prefix removed? Isn't it a return variable?

Reply:
We usually set "ret_" prefix when the value is unconditionally set on success. But, now this function sets conditionally. So, I dropped the prefix.